apps/folderlaunch/types.d.ts

```diff
@@ -21,7 +21,6 @@ type Config = {
     apps: {                         // Saved info for each app
         [key: string]: {
             folder: FolderList,     // Folder path
-            fast: boolean,          // Whether the app should be fast launched
```

Comment:
@bruceblore hope these changes are ok with you - to explain this one: with the typings added to `readJSON`, typescript now points out that when we [create an app config](https://github.com/espruino/BangleApps/blob/57f01d295141cf5d6e6c085315526ae21580c5d6/apps/folderlaunch/configLoad.ts#L128-L131) we weren't adding the `fast` property. I checked and it doesn't seem used, so I've taken it out here

Reply:
Yes, that's OK to remove. (I should have removed it myself a while ago.) Originally, I intended folderlaunch to allow for manually setting apps to be fast loaded. However, someone pointed out that fastload utils existed, and therefore that my own manually-specified fast-loading feature did not make sense. So I removed the feature, but I guess I just forgot to remove this reference to it from the types.